docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,28 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/material-ui/react-modal/) and [`Popper`](/material-ui/react-popper/) components.
+`Portal` is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
```

Comment:
It think that we can keep the content more "lightweight" (easier to read, less style change interruption) with:

```suggestion
Portal is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
```

We don't need to really React in this case?

Reply:
I think it's helpful here to differentiate between `Portal` (the component) vs. any generic portal generated by the `createPortal()` API.

Reply:
The distinction already seems clear from "Portal is a utility _component_"

Reply:
> Looking at the Material UI docs, we don't seem to be consistent in the use of backticks for component names. It does seem the be the predominant style, but it isn't necessarily better. The capitalisation seems sufficient, along with an indefinite article: "a portal" vs. "Portal"

I thought we had established a house rule for this in the Style guide, but I'm not seeing anything about it now that I'm looking through it. Must have been deep in some PR comments somewhere. 

In the Base docs, I've been trying to stick to a consistent format of backticks and capitalization anytime I'm referring to the component specifically, in the interest of being as clear and precise as possible.

In any case, this is a question that applies to all of the Base docs, so I propose that we leave it as is for now, and continue this conversation about formatting to my next PR which will be a final pass covering all of these pages.